roles/openshift_logging_mux/defaults/main.yml

```diff
@@ -9,8 +9,10 @@ openshift_logging_mux_namespace: logging
 
 ### Common settings
 openshift_logging_mux_nodeselector: "{{ openshift_hosted_logging_mux_nodeselector_label | default('') | map_from_pairs }}"
-openshift_logging_mux_cpu_limit: 500m
-openshift_logging_mux_memory_limit: 1Gi
+openshift_logging_mux_cpu_limit: "{{ openshift_hosted_logging_mux_cpu_limit | default('500m') }}"
```

Comment:
Sorry, please bear with me...   With having openshift_logging_mux_memory_limit: "1Gi" in defaults/main.yml, when I set this in my inventory file, the value 4Gi is picked up?
openshift_logging_mux_memory_limit=4Gi

And let me reset {mux_cpu_limit, mux_memory_limit} = {500m, 2Gi} since I was told the ElasticSearch's default is much higher {1000m, 8Gi}.